nevergrad/optimization/differentialevolution.py

```diff
@@ -286,6 +296,7 @@ def __init__(
         popsize: tp.Union[str, int] = "standard",
         propagate_heritage: bool = False,  # experimental
         multiobjective_adaptation: bool = True,
+        high_speed: bool = False,
```

Comment:
please update the docstring + I am not sure "highspeed" is the most descriptive name

Reply:
Done. High-Speed is my best guess for now.
For example FastGA became a famous name whereas it does not say much about what it's doing :-)